entry/src/main/js/default/pages/index/index.hml

```diff
@@ -0,0 +1,98 @@
+<element src="../../../../../../../smartwash/js/index.hml" name="test"></element>
+<div class="container">
+    <div class="full">
+        <div class="timer">
+            <image class="level_bg" src="/common/images/level_bg.png"></image>
+            <image class="Timmer_bg" src="/common/images/Timer_bg.png"></image>
+            <div class="centered"><text>0:30</text></div>
+            <progress class="min-progress" type="ring" percent= "40" ></progress>
+            <image class="indication_point" src="/common/images/indication_point.png"></image>
+
+            <image src="/common/images/fill.png"class="fill"></image>
```

Comment:
All such options displayed in UI, please maintain as array in js and use that to display in hml, instead of directly hardcoding over here